src/vm/docs/mod.rs

```diff
@@ -393,6 +393,28 @@ principal, and executes `expr` with that context. It returns the resulting value
     example: "(as-contract (print tx-sender)) => 'CTcontract.name"
 };
 
+const GET_BLOCK_INFO_API: SpecialAPI = SpecialAPI {
+    input_type: "BlockInfoPropertyName, BlockHeightInt",
+    output_type: "buff|int",
+    signature: "(get-block-info prop-name block-height-expr)",
+    description: "The `get-block-info` function fetches data for a block of the given block height. The 
+value and type returned is determined by the specified property name. If the provided block height integer does
+not correspond to an existing block, the function returns Void. The currently available property names 
+are `time`, `header-hash`, and `vrf-seed`. 
+
+The `time` property returns an integer value of the block header time field. This is a Unix epoch timestamp in seconds 
+which roughly corresponds to when the block was mined. Warning: this does not increase monotonically with each block
+and block times are accurate only to within two hours. See BIP113 for more information. 
+
+The `header-hash` property returns a 32-byte buffer. The `vrf-seed` property returns a 32-byte buffer. 
+",
+    example: "(get-block-info time 10) => 1557860301
+(get-block-info time (- 100)) => Void
```

Comment:
I think this should cause the contract to abort if you pass an invalid argument.